cmd/restore.go

```diff
@@ -44,6 +44,8 @@ func NewRestoreCommand() *cobra.Command {
 
 	command.PersistentFlags().Uint("concurrency", 128,
 		"The size of thread pool that execute the restore task")
+	command.PersistentFlags().Uint64("rateLimit", 128,
```

Comment:
```suggestion
	command.PersistentFlags().Uint64("ratelimit", 1024,
```

Please improve code coverage by choosing a test and adding a ratelimt flag,

Reply:
how about setting default value to 0 without ratelimit flag?